frontend/src/components/Client/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,21 @@
+const Dropdown = ({prefix, sortOptions = [], handleOptions, defaultValue}) => {
+  const options = sortOptions.map((option, i) => (
+    <option value={option.value} key={i}>{option.display}</option>
+  ))
+
+  return (
+    <div className="relative form-input py-0 w-48 m-4 md:my-0 border-gray-600 px-1 hover:border-indigo-400/80 ease-in duration-150 bg-transparent">
+      <span className="absolute left-2 bottom-[10px] z-0 pointer-events-none">{prefix && `${prefix}:`} </span>
+      <select
+        className="bg-transparent focus:outline-none w-full pl-8 font-medium py-2"
+        required
+        onChange={handleOptions}
+        defaultValue={defaultValue}
+      >
+        {options}
+      </select>
+    </div>
```

Comment:
I think visually, there should be a bit more space between the prefix and the defaultValue of the select input. 
We can achieve this by:

1. Replacing "relative" in line 7 with "flex justify-around items-center" and "py-0" with "py-2"
2. Leave only "pointer-events-none" in line 8
3. Leave only "focus:outline-none font-medium" in line 10

Reply:
I initially did this, but decided against it because it disabled the portion of the bar that was not associated with the `<select>` itself. The `prefix` section is unclickable because its not part of the `<select>` section, but is wrapped in the same parent border which makes for a really weird experience.

Here is a screenshot - the red represents the dead zone that cannot be clicked. The yellow is the only clickable section that will show the dropdown
![image](https://user-images.githubusercontent.com/5695484/230803195-4b474f27-6994-4bdd-aea2-ac958cc4186b.png)

The way that I implemented it here with relative and absolute sections allows the whole section to be treated as a dropdown-
![image](https://user-images.githubusercontent.com/5695484/230803307-fa15e437-f126-4464-9f7b-1715a2ab3b3f.png)

Reply:
Another option I was proposing was to add an onClick listener to the parent div, but that doesn't work either because apparently `<select>`  does not have an `open()` method, only a `focus()`[ method which doesn't actually open the select menu](https://stackoverflow.com/questions/63382309/how-can-i-trigger-opening-a-select-forms-options-on-click-of-another-button-in). The suggestion here is to use a library like react-select. We have this installed so this is possible, but I think may be out of the scope of this ticket. I can include it in the next ticket as part of adding functionality to the actual dropdown.

Reply:
The select functions better when the entire bordered element is clickable as you implemented. I think for now you can just space out the prefix and defaultValue a bit more to temporarily fix the overlapping issue if the prefix were to be longer. Perhaps add text-center or text-end to the select element.

Reply:
This has been updated to be centered 👍 
![image](https://user-images.githubusercontent.com/5695484/231038612-8b30ff48-920d-4919-b2da-334a3392991a.png)